livesync_citadel/indico_livesync_citadel/migrations/20191010_1337_51e3862bad1d_create_table_searchapp_id_map.py

```diff
@@ -0,0 +1,45 @@
+"""Create table searchapp_id_map
+
+Revision ID: 51e3862bad1d
+Revises:
+Create Date: 2019-10-10 13:37:57.440636
+"""
+
+from __future__ import unicode_literals
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.sql.ddl import CreateSchema, DropSchema
+
+from indico.core.db.sqlalchemy import PyIntEnum, UTCDateTime
+
+from indico_livesync_citadel.models.search_id_map import EntryType
+
+
+# revision identifiers, used by Alembic.
+revision = '51e3862bad1d'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.execute(CreateSchema('plugin_livesync_citadel'))
+    op.create_table('searchapp_id_map',
```

Comment:
I don't like the name of this table. Any ideas? This is the mapping between an Indico object and the ID of an ES document.

Reply:
`es_id_map`?